userguide/image_classifier/README.md

```diff
@@ -17,7 +17,7 @@ The [Kaggle Cats and Dogs Dataset](https://www.microsoft.com/en-us/download/deta
 ```python
 import turicreate as tc
 
-# Load images (Note: you can ignore 'Not a JPEG file' errors)
+# Load images (Note:'Not a JPEG file' errors are simply warnings, meaning those files will simply be skipped)
```

Comment:
The word `simply` appears twice in this sentence.

Reply:
Updated the PR not to use a qualifier